### ISSUE TITLE
brew install fails, points to missing v0 files rather than the new v1 files

### ISSUE THREAD

**Issue by khgitting** [theory/homebrew-sqitch#32]
```
brew tap theory/sqitch
brew install sqitch
```
results in 

```
...
==> Downloading http://api.metacpan.org/source/DWHEELER/App-Sqitch-0.9995/META.json

curl: (22) The requested URL returned error: 410 Gone
Error: Failed to download resource "sqitch_dependencies"
Download failed: http://api.metacpan.org/source/DWHEELER/App-Sqitch-0.9995/META.json
```

If you navigate to that url in the browser, you see that v1 resources should be used instead:
```
MetaCPAN v0 API has been has been shut down!

See the MetaCPAN v1 API should be used instead.
```

**Comment by theory** [comment]
Thank you! Should be fixed in 603eabe45.

**Comment by khgitting** [comment]
Dang, nice turn around time 👍
